corehq/apps/receiverwrapper/tests/test_submissions.py

```diff
@@ -20,11 +20,9 @@
 from corehq.util.test_utils import TestFileMixin, softer_assert
 
 
-class SubmissionTest(TestCase):
-    maxDiff = None
-
+class BaseSubmissionTest(TestCase):
     def setUp(self):
```

Comment:
almost all of setup and teardown could be done at the class level (except perhpas for deleting the forms and cases in teardown). This would make the tests much faster.

Reply:
Tried that and all tests started failing. Must have done something incorrectly but not up for some test debugging right now so going to leave this as-is